imdigitalashish.md

```diff
@@ -0,0 +1,46 @@
+---
+name: "@imdigitalashish"
+project: "Zenith"
+---
+
+
+# Zenith
+
+## Summary
+
+It is a home automation tool that resembles IRON man Jarvis Assistant. I will create this from groud up including all codes and hook
+it up with hardware to automate my common tasks and open source it so everyone can use it and fullfill their IRON man dream !
+
+## Plan
+
+I will use Raspberry Pi 4 as main controller of the board running RUST (for key performance areas) and Python (for AI / ML). With that
+I proceed by connecting to hardware by wifi or wires or bluetooth to control and get data and make variety of features such as intruder detection (servo motor), turn on lights, integrate it with ChatGPT and my own chat model to communicate, make a beautifull GUI'S, create a server to communicate over call through Azure and control my room from anywhere in the world and to store stuff by just saying "JARVIS SAVE THIS FILE" to a harddrive using raspberry PI ( just like airdrop ) and incrementally add features time to time.
+
+Features that I will be making:
+
+1) Apple Airdrop feature with Harddrive and Raspberry PI
```

Comment:
what does this look like and what do you have to do to make it work?

Reply:
So sometimes there are cases in which we have to share or store files in our computer urgently or we need to share file to a person where there is only browser and no whatsapp etc. There are services like wetransfer etc but there is a lot of friction ! So I will use Raspberry PI connected with RUST backend to store files very fast and it will generate a shortlink that will redirect to the file and we can also use alias to store that file so we can share and store really fast !

Reply:
If you want I can share the block diagram of the system design !

Reply:
ooo cool! yes that would be great! can you please add that within your file?

Reply:
Sure, Added to the file !